getting-started/trino/README.md

```diff
@@ -21,6 +21,12 @@
 
 This getting started guide provides a `docker-compose` file to set up [Trino](https://trino.io/) with Apache Polaris. Apache Polaris is configured as an Iceberg REST Catalog in Trino. 
 
+## Build Polaris Image
+Build Polaris Image while Docker is running
+```
+./gradlew :polaris-quarkus-server:assemble -Dquarkus.container-image.build=true
```

Comment:
General question (not about this PR specificaly): should not we rename `polaris-quarkus-server` to `polaris-server` now ? @adutra thoughts ?

Reply:
Not opposed to a global renaming and removing the `quarkus` token from all module names 👍